BREAKING_CHANGES.md

```diff
@@ -0,0 +1,7 @@
+Breaking Changes
```

Comment:
Alternatively, can just use the existing change log? Although I don't see us populating it manually for every change. We might want to automate it during the publishing step in that case.

Reply:
I'd lean towards doing it in the existing changelog, I have an open PR for docs site fixes that adds a description that we won't update changelog for every version but will try to for breaking changes so I like this addition.